File: python/linhasql.py
# Import da biblioteca:
import mysql.connector
import time

start = time.time()

# Conexão com o banco de dados:
conexao = mysql.connector.connect(user='root', password='', host='localhost', database='testemauro200422')

# Cria um cursor:
cursor = conexao.cursor()

# Executa o comando:
for i in range(100):
  query = f"INSERT INTO python(placa, nome_dono) VALUES ('teste {i}', 'Joao {i}')"
  with conexao.cursor() as cursor:
    cursor.execute(query)
    conexao.commit()

# Finaliza a conexão
conexao.close()

end = time.time()

print((end - start)*1000)

File: js/linhasql.js
const mysql = require('mysql');
console.time('#time');

var con = mysql.createConnection({
    host: "localhost",
    user: "root",
    password: "",
    database: "testemauro200422"
});

con.connect(function(err) {
    if (err) throw err;
    console.log("Connected!");
    try {
        for (i = 0; i < 100; i++) {
            let query = `INSERT INTO js(placa , nome_dono) VALUES('test ${i}', 'Joao ${i}')`
            con.query(query, function(err, result) {
                if (err) throw err;
            });
        }
    } finally {
        con.end();
    }
});

console.timeEnd('#time');